examples/monaco-graphql-react-vite/src/App.tsx

```diff
@@ -1,69 +1,67 @@
-import React, { useEffect } from 'react';
+import { ReactElement, useEffect, useRef, useState, StrictMode } from 'react';
 import { getIntrospectionQuery, IntrospectionQuery } from 'graphql';
 import { Uri, editor, KeyMod, KeyCode, languages } from 'monaco-editor';
 import { initializeMode } from 'monaco-graphql/src/initializeMode';
-import { createGraphiQLFetcher } from '@graphiql/toolkit';
+import { createGraphiQLFetcher, SyncFetcherResult } from '@graphiql/toolkit';
 import * as JSONC from 'jsonc-parser';
 import { debounce } from './debounce';
 
 const fetcher = createGraphiQLFetcher({
-  url: 'https://api.spacex.land/graphql/',
+  url: 'https://countries.trevorblades.com',
 });
 
 const defaultOperations =
   localStorage.getItem('operations') ??
-  `
-# cmd/ctrl + return/enter will execute the op,
+  `# cmd/ctrl + return/enter will execute the op,
 # same in variables editor below
 # also available via context menu & f1 command palette
 
-query($limit: Int!) {
-    payloads(limit: $limit) {
-        customer
-    }
-}
-`;
+query($code: ID!) {
+  country(code: $code) {
+    awsRegion
+    native
+    phone
+  }
+}`;
 
 const defaultVariables =
   localStorage.getItem('variables') ??
-  `
- {
-     // limit will appear here as autocomplete,
-     // and because the default value is 0, will
-     // complete as such
-     $1
- }
-`;
-
-const getSchema = async () =>
-  fetcher({
+  `{
+  // 'code' will appear here as autocomplete,
+  // and because the default value is 0, will
+  // complete as such
+  $1
+}`;
+
+async function getSchema(): Promise<SyncFetcherResult> {
+  return fetcher({
     query: getIntrospectionQuery(),
     operationName: 'IntrospectionQuery',
   });
+}
 
-const getOrCreateModel = (uri: string, value: string) => {
+function getOrCreateModel(uri: string, value: string): editor.ITextModel {
   return (
     editor.getModel(Uri.file(uri)) ??
     editor.createModel(value, uri.split('.').pop(), Uri.file(uri))
   );
-};
+}
 
-const execOperation = async function () {
+async function execOperation(): Promise<void> {
   const variables = editor.getModel(Uri.file('variables.json'))!.getValue();
   const operations = editor.getModel(Uri.file('operation.graphql'))!.getValue();
   const resultsModel = editor.getModel(Uri.file('results.json'));
-  // @ts-expect-error
   const result = await fetcher({
     query: operations,
-    variables: JSON.stringify(JSONC.parse(variables)),
+    variables: JSONC.parse(variables),
```

Comment:
here was a mistake, we should pass object and not string

Reply:
hm,,, this used to work! but perhaps monaco-graphql changed